src/Symfony/Component/Workflow/Tests/WorkflowTest.php

```diff
@@ -313,8 +313,9 @@ public function testGetEnabledTransitions()
 
         $subject->marking = array('c' => 1, 'e' => 1);
         $transitions = $workflow->getEnabledTransitions($subject);
-        $this->assertCount(1, $transitions);
-        $this->assertSame('t5', $transitions[0]->getName());
+        $this->assertCount(2, $transitions);
+        $this->assertSame('t2', $transitions[0]->getName());
```

Comment:
Hi @sebdec this change is not valid. To apply t2, the subject must be in all froms, in this case "b" and "c", but only "c" is marked.